### PR TITLE
Adds GitHub Actions CI/CD

### LINKED ISSUE #5: Setup CI/CD migration from gitlab ci -> github actions

### PR DESCRIPTION
**Pull request by marshallmcdonnell**
Closes #5 

Work includes:
  - Switches from code.ornl.gov -> Docker Hub base container images
  - Adds GitHub Actions CI/CD
  - Cleanup README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
 
 ## Getting Started
 
+First, you need to [login to the GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
+
 The fastest way to spin up a working setup is to use docker-compose top-level file:
 
 ```
-docker login code.ornl.gov:4567
 docker-compose up
```
Comment (gecage952) [inline README.md]: Just wondering, I know the newer version of compose (v2 I think) uses the space instead of hyphen. It might be worth mentioning here?

**Review by gecage952 — APPROVED** [review]
Runs for me locally and checks pass. My comment above isn't really critical or anything.

File: README.md (hunk 1 above)
Comment (marshallmcdonnell) [reply README.md]: Ah, good call, yes!

On our ORNL infrastructure (gitlab-runners) we mainly have v1 of docker compose.
Yet, for this project, we should probably go ahead and move over to v2 for now (easy work.

I'll make this change.
Thanks :+1:

Comment (marshallmcdonnell) [reply README.md]: Done! Sorry for delay. Tested locally and working for me

**Review by Lance-Drane — APPROVED** [review]
Everything LGTM

File: README.md (hunk 1 above)
Comment (Lance-Drane) [reply README.md]: I don't think the compose file needs to be modified at all, it probably is as simple as removing the hyphen _and_ making sure that you have the compose plugin installed.

There could be something similar happening with Docker buildx replacing the old `build` command, which is currently deprecated.

Comment (marshallmcdonnell) [reply README.md]: Thanks, I just quickly changed them all not realizing that is talking about the file :disappointed: 

I'll fix in coming PR

Comment (marshallmcdonnell) [reply README.md]: Commit that will fix it: https://github.com/smart-spectral-matching/ssm-service-catalog/commit/7a6abce35959d39d36f9a0604dca9fb4a6ecce68

**Pull request merged**